dist/.magento.env.yaml

```diff
@@ -336,6 +336,21 @@
 #                    initStatements: SET NAMES utf8                                                                   #
 #                    active: 1                                                                                        #
 #######################################################################################################################
+# RESOURCE_CONFIGURATION - use to configure mapping of resource name to a connection.                                 #
```

Comment:
Probably need to clarify that it is DB resource connection. (as in the config name as in description)